lib/rdoc/generator/template/darkfish/css/rdoc.css

```
@@ -292,6 +292,14 @@ main {
   main {
     margin-left: var(--sidebar-width);
   }
+
+  .table-of-contents main {
+    margin-left: 20em;
+  }
+
+  #navigation-toggle {
+    display: none !important;
```

I don't think this `!important` is necessary?
```suggestion
    display: none;
```